en/installation-guide/database-initialization.rst

```diff
@@ -86,6 +63,24 @@ Restore the latest data model that also includes demo data:
 
 There are now 4 schemas in the database (qgep_od, qgep_sys, qgep_vl, qgep_import)
 
+Create  minimal roles and access
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: The QGEP roles are defined in the https://github.com/QGEP/datamodel/blob/master/12_roles.sql file. It is recommended to use these when using QGEP in a production environment.
+
+* Create a new group role (preferably named `qgep`, but you are free to choose)
+
+* Create a new login role (`qgepuser` for example) and make it a member of the `qgep` group
+
+You can do this graphically or by simply opening a pgAdmin :ref:`SQL-query` window and type the following:
+
+::
+
+     CREATE GROUP qgep;
+     CREATE ROLE qgepuser LOGIN;
+     GRANT qgep TO qgepuser;
```

Comment:
I wonder if we should leave this paragraph.

- For a "quick and dirty" demo-setup, I don't care to use the super user (postgres or similar) to ignore any access right troubles
- For a production installation, 12_roles.sql (or a derived, organisation specific setup) should be used

This approach here oversimplifies a more complex topic with remaining pitfalls (namely forgetting to grant correct access rights).

Reply:
l would be in favor of removing this approach which is very bad if set in production (the standard user would have too much permissions in that case)

Reply:
Can you adjust this or shall we put that in a separate PR?

Reply:
I removed dirty approach in 24c1012